### DISCUSSION TITLE
How to submit a streaming task？

### DISCUSSION THREAD

**Discussion by LemonLiTree** [bytedance/bitsail#184] [Q&A]
It is OK to test by Local Flink Session，but this is a batch task，so how to submit a streaming task。
jdbc(mysql) source -> doris sink

**Comment by garyli1019** [comment]
hi @LemonLiTree , sorry we didn't found this question in the discussion tab. We usually answer question in the Issues session. 
Do you mean consuming mysql Binlog and write to doris? We are currently working on support the mysql cdc source and we will have a alpha version by the end of this month.

**Reply by LemonLiTree** [reply]
ok
